src/generate/utils.ts

```diff
@@ -228,22 +249,23 @@ export const getValue = <T>(name: model.ValueField<T>): string => {
 type propertyFilter = (propertyName: string) => boolean;
 
 /**
- * Get properties of the DTO (inherited and linked) after applying the given filter criteria
+ * Get properties of a node (inherited and linked) after applying the given filter criteria
+ *
+ * @param node - An AMF node
+ * @param propertyFilter - predicate to be used to filter properties
  *
- * @param dtoTypeModel - AMF model of the dto
- * @param propertyFilter - function to filter properties based on certain criteria
  * @returns The filtered list of properties
  */
 export const getFilteredProperties = (
-  dtoTypeModel: model.domain.NodeShape | null | undefined,
+  node: model.domain.NodeShape | null | undefined,
```

Comment:
Will `null` or `undefined` ever actually be passed to the function?

Reply:
Probably not. I didn't know the reason it was added for so didn't bother updating it.